### PR TITLE
[wicket] Trigger updates and report installed versions for SP and RoT

### PR DESCRIPTION
**Pull request by andrewjstone**
This is a couple of commits for independent features. Future commits will add a warning popup for triggering updates and also provide progress on status and event logs.

### CODE REVIEW

File: wicket/src/runner.rs [hunk 1]
```diff
@@ -22,6 +22,7 @@ use tui::backend::CrosstermBackend;
 use tui::Terminal;
 
 use crate::ui::Screen;
+use crate::wicketd;
```
Comment (andrewjstone) [inline wicket/src/runner.rs:25]: This should probably move into the next line as `self`

File: wicket/src/runner.rs [hunk 2]
```diff
@@ -205,7 +206,11 @@ impl Runner {
             Action::Redraw => {
                 self.screen.draw(&self.state, &mut self.terminal)?;
             }
-            Action::Update(_component_id) => todo!(),
+            Action::Update(component_id) => {
+                self.wicketd.tx.blocking_send(
+                    wicketd::Request::StartUpdate(component_id),
```
Comment (jgallagher) [inline wicket/src/runner.rs:211]: Just double-checking terminology - a "component" here identifies a sled/switch/psc and not a component within a sled, right?

**Review by jgallagher — APPROVED** [review]
LGTM!

File: wicket/src/runner.rs (hunk 2 above)
Comment (andrewjstone) [reply wicket/src/runner.rs:211]: Correct.

File: wicket/src/runner.rs (hunk 1 above)
Comment (andrewjstone) [reply wicket/src/runner.rs:25]: Fixed in a follow up.

**Pull request merged**